src/_data/whats-new.yml

```diff
@@ -1,11 +1,48 @@
 title: Whats New on Devdocs
-description: |
+description: 
```

Comment:
There was a reason why this pipeline was there. Whatsnew updates can update only `updated` and `etnries`.